templates/inactive_reviewer_needinfo.txt

```diff
@@ -0,0 +1,19 @@
+The following {{ plural('patch is', revisions, 'patches are') }} waiting for review from an inactive reviewer:
+
+| ID  | Title |  Author | Reviewer Status |
+| --- | :---- |  ------ | --------------- |
+{% for rev in revisions -%}
+{% for reviewer in rev['reviewers'] -%}
+{% if rev['reviewers'][0] == reviewer -%}
+| [D{{ rev['rev_id'] }}](https://phabricator.services.mozilla.com/D{{ rev['rev_id'] }}) |  {{ rev['title'] | e }} | [{{ rev['author']['nick'] }}](https://bugzilla.mozilla.org/user_profile?user_id={{ rev['author']['id'] }})  |
+{%- else -%}
+| | | |
+{%- endif -%}
+
+[{{ reviewer['phab_username'] }}](https://phabricator.services.mozilla.com/p/{{ reviewer['phab_username'] }}/): {{ reviewer["status_note"] }} |
+{% endfor -%}
+{% endfor %}
+
+:{{ nicknames }}, could you please find another reviewer or {% if has_old_patch -%} abandon the patch if it is no longer relevant {%- endif%}?
```

Comment:
The "or" should be after the if, right?

Reply:
Yes, nice catch. Fixed in 2eb08c57081f8aa327c4d5e6a31e27ef26be7e21.